sdks/python/apache_beam/io/lyft/kafka.py

```diff
@@ -163,7 +167,7 @@ def to_runner_api_parameter(self, _unused_context):
   @PTransform.register_urn("lyft:flinkKafkaSink", None)
   def from_runner_api_parameter(_unused_ptransform, spec_parameter, _unused_context):
     logging.info("kafka spec: %s", spec_parameter)
-    instance = FlinkKafkaInput()
+    instance = FlinkKafkaSink()
```

Comment:
I am still so confused as to how this was even working before especially because `topics` in `FlinkKafkaInput` is a list.

Reply:
I am with you! honestly, it beats my imagination at this point.